application-forum-ui/src/main/resources/ForumCode/Actions.xml

```diff
@@ -65,26 +67,34 @@
         $response.setStatus(403)
       #end
     #end
-  #elseif("$!request.action" == "addtopic")
+  #elseif("$!request.action" == 'addtopic')
     #if("$!request.title.trim()" != '')
       #if("$!request.space" != '')
-        #set($newTopicName = $request.page)
```

Comment:
Nevermind, I'll use the same approach as before. Topic spaces will have unique names like Topic_0